time odom_stamp
time gps_stamp
float32 x  
float32 y 
float32 z  
float32 gps_x
float32 gps_y
float32 qx  
float32 qy  
float32 qz  
float32 qw  
float32 battary_velocity
bool heart_beat # 心跳数据，表示planner是否还活着
